### PR TITLE
add dop/autotest/testplan proto

### PR DESCRIPTION
**Pull request by ZLValien**
add autotest-testplan proto

### CODE REVIEW

File: core/dop/autotest/testplan/autotest-plan.proto [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+package erda.core.dop.autotest.testplan;
+option go_package = "github.com/erda-project/erda-proto-go/core/dop/autotest/testplan/pb";
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+service TestPlanService {
+  rpc UpdateTestPlanV2ByHook (TestPlanV2UpdateByHookRequest) returns (TestPlanV2UpdateByHookResponse)  {
+    option (google.api.http) = {
+      post: "/api/actions/autotest-plan-execute-callback",
```
Comment (sfwn) [inline core/dop/autotest/testplan/autotest-plan.proto:11]: /api/autotests/actions/execute-callback

File: core/dop/autotest/testplan/autotest-plan.proto [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    };
+  }
+}
+
+// TestPlanV2UpdateByHooksRequest update testPlan by hook (post /api/actions/autotest-plan-execute-callback )
+message TestPlanV2UpdateByHookRequest{
```
Comment (sfwn) [inline core/dop/autotest/testplan/autotest-plan.proto:17]: what is v2

**Review by sfwn — CHANGES_REQUESTED** [review]

File: core/dop/autotest/testplan/autotest-plan.proto (hunk 2 above)
Comment (ZLValien) [reply core/dop/autotest/testplan/autotest-plan.proto]: V2 is old name.
refactor to TestPlanUpdateByHookRequest

File: core/dop/autotest/testplan/autotest-plan.proto (hunk 1 above)
Comment (ZLValien) [reply core/dop/autotest/testplan/autotest-plan.proto]: /api/autotests/actions/plan-execute-callback

**Review by sfwn — APPROVED** [review]

**Pull request merged**